core/src/main/java/org/infinispan/xsite/BackupSenderImpl.java

```diff
@@ -123,13 +123,14 @@ private static boolean isCommunicationError(Throwable throwable) {
 
    private enum BackupFilter {KEEP_1PC_ONLY, KEEP_2PC_ONLY, KEEP_ALL}
 
-   public BackupSenderImpl(String localSiteName) {
-      this.localSiteName = localSiteName;
+   public BackupSenderImpl() {
    }
 
    @Start
    public void start() {
+      transport.checkCrossSiteAvailable();
```

Comment:
can't we just do this check inside the localSiteName and cal the localSiteName method here ?

Reply:
wouldn't that be bad for the console? 
each time you invoke `localSiteName()` it will throw an exception if JGroups isn't configured with RELAY2

Reply:
ok